File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/OccupancyRateFee.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses

    Public Class OccupancyRateFee

        <JsonProperty("billable_currency")>
        Public Property TotalInBillableCurrency As OccupancyRateAmount

        <JsonProperty("request_currency")>
        Public Property TotalInRequestCurrency As OccupancyRateAmount

        <JsonProperty("scope")>
        Public Property FeeScope As String

        <JsonProperty("frequency")>
        Public Property FeeFrequency As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/OccupancyRateFee.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses
{
    using Newtonsoft.Json;

    public class OccupancyRateFee
    {

        [JsonProperty("billable_currency")]
        public OccupancyRateAmount TotalInBillableCurrency { get; set; }

        [JsonProperty("request_currency")]
        public OccupancyRateAmount TotalInRequestCurrency { get; set; }

    }

}